src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Extensions/OpenApiEndpointRouteBuilderExtensionsTests.cs

```diff
@@ -59,7 +59,7 @@ public async Task MapOpenApi_ReturnsRenderedDocument()
         context.Response.Body = responseBodyStream;
         context.RequestServices = serviceProvider;
         context.Request.RouteValues.Add("documentName", "v1");
-        var endpoint = builder.DataSources.First().Endpoints.First();
+        var endpoint = builder.DataSources.First().Endpoints[0];
```

Comment:
Is moving from `First()` to the indexer a result of XUnit analyzers here or...?

Reply:
Nope! This change looks good. I just wanted to verify that it was coming from the analyzer. I believe similar warnings have been addressed in other files.